`Test-VcfPasswordManagementPrereq ` shows errors if multiple version are installed

### Code of Conduct

- [X] I have read and agree to the [Code of Conduct](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/blob/main/CODE_OF_CONDUCT.md).
- [X] Vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue initial description to help the maintainers prioritize.
- [X] Do not leave "+1" or other comments that do not add relevant information or questions.
- [X] If you are interested in working on this issue or have submitted a pull request, please leave a comment.

### VMware Cloud Foundation

5.1.0.0

### PowerShell Version

7.4.1

### PowerCLI Version

13.2.1

### Module Version

1.7.1.1004

### PowerValidatedSolutions Version

2.9.0.1028

### PowerVCF Version

2.4.1.1000

### Guest Operating System

Windows server 2019

### Environment Details

_No response_

### Description

Test-VcfPasswordManagementPrereq could not handle multiple installed version of required packages

### Error or Debug Output

```
PS F:\PlatformTools> Test-VcfPasswordManagementPrereq
 [03-07-2024_14:18:53] [INFO] VMware.vSphere.SsoAdmin 1.3.9 is installed version and meets the minimum required version of VMware.vSphere.SsoAdmin 1.3.9.
 [03-07-2024_14:18:53] [ERROR] PowerVCF 2.4.1.1000 2.4.0.1011 2.3.0.1004 1.2.5 is installed. Install PowerVCF 2.4.1 or higher.
 [03-07-2024_14:18:53] [ERROR] PowerValidatedSolutions 2.9.0.1028 2.7.0.1013 is installed. Install PowerValidatedSolutions 2.8.0 or higher.
PS F:\PlatformTools>
```

### Expected Behavior

Take into account latest installed version

### Actual Behavior

Error - see above

### Steps to Reproduce

Run "Test-VcfPasswordManagementPrereq" on a machine with several version of required packages

### Log Fragments and Files

_No response_

### Screenshots

_No response_

### References

_No response_

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -16,6 +16,7 @@ Bugfix:
 
 - Fix for missing account lockout policy data for SDDC Manager and vCenter Server. [GH-160](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/pull/160)
 - Fix for `VMware.PowerCLI` module name not being mentioned in the required modules list of the manifest file. [GH-170](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/pull/170)
+- Fix for `Test-VcfPasswordManagementPrereq` not working while multiple module versions were present. [GH-174](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/pull/174)
 
 ## v1.7.1
 

--- a/VMware.CloudFoundation.PasswordManagement.psd1
+++ b/VMware.CloudFoundation.PasswordManagement.psd1
@@ -11,7 +11,7 @@
     RootModule        = '.\VMware.CloudFoundation.PasswordManagement.psm1'
 
     # Version number of this module.
-    ModuleVersion     = '1.7.2.1003'
+    ModuleVersion     = '1.7.2.1004'
 
     # Supported PSEditions
     # CompatiblePSEditions = @()

--- a/VMware.CloudFoundation.PasswordManagement.psm1
+++ b/VMware.CloudFoundation.PasswordManagement.psm1
@@ -9948,8 +9948,9 @@ Function Test-VcfPasswordManagementPrereq {
             $installedModule = Get-Module -ListAvailable -Name $moduleName
 
             if ($installedModule) {
-                $installedVersion = $installedModule.Version
-                if ($installedVersion -lt $requiredVersion) {
+                $installedVersion = $installedModule.Version | Sort-Object -Descending
+                $installedVersion = $installedVersion[0]
+                if ($installedVersion -lt $requiredVersion) { 
                     $message = "$($moduleName) $($installedVersion) is installed. Install $($moduleName) $($requiredVersion) or higher."
                     Show-PasswordManagementOutput -type ERROR -message $message
                 } elseif ($installedVersion -ge $requiredVersion) {

